plone/session/plugins/session.py

```diff
@@ -176,9 +178,12 @@ def extractCredentials(self, request):
             return creds
 
         try:
-            creds["cookie"] = binascii.a2b_base64(
+            cookie = binascii.a2b_base64(
                 request.get(self.cookie_name)
             )
+            if not six.PY2 and isinstance(cookie, bytes):
+                cookie = safe_unicode(cookie)
```

Comment:
@ale-rt maybe this condition can go away, and always use `safe_unicode`, what do you think?

Reply:
Will not work, I think cookie has to be bytes.

Reply:
I had this environment just on my other laptop before Apple send a firmware update and make it stop working... so I can't reproduce it again right now